File: app/src/main/java/me/sunzheng/mana/home/onair/wrapper/AirWrapper.java

package me.sunzheng.mana.home.onair.wrapper;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang3.builder.ToStringBuilder;

import java.util.List;

public class AirWrapper {

    @SerializedName("data")
    @Expose
    private List<BangumiModel> data = null;

    public List<BangumiModel> getData() {
        return data;
    }

    public void setData(List<BangumiModel> data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return ToStringBuilder.reflectionToString(this);
    }

}

File: app/src/main/java/me/sunzheng/mana/core/net/v2/wrappers/AirWrapper.kt
package me.sunzheng.mana.core.net.v2.wrappers

import com.google.gson.annotations.SerializedName
import me.sunzheng.mana.core.net.v2.database.BangumiModel


data class AirWrapper(
    @SerializedName("data")
    val data: List<BangumiModel>?
)